src/solvers/sat/external_sat.cpp

```diff
@@ -0,0 +1,188 @@
+/*******************************************************************\
+
+Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
:no_entry_sign: This is currently saying that the copyright is either owned by Amazon or someone who isn’t Amazon, which is pretty confusing. The all rights reserved bit is also a bit weird, if you are making a contribution it must be under the same license as cbmc itself (see https://github.com/diffblue/cbmc/blob/develop/LICENSE).

Also instead of heaving a header like this please specify who the author or copyright owner is via doxygen using syntax like

```
/// \file
/// \author author-names-go-here
```

Reply:
🐰